src/components/toolbar/toolbar.html

```diff
@@ -0,0 +1 @@
+<ng-content></ng-content>
```

Comment:
When the template of a component is just `<ng-content>`, it can be a `@Directive` (no template) instead of a `@Component`